themes/digital.gov/src/scss/new/styles.scss

```diff
@@ -175,6 +175,7 @@
 
 @forward 'shortcodes/accordion';
 @forward 'shortcodes/box';
+@forward 'shortcodes/checkbox';
```

Comment:
Should we name this `checklist` instead of `checkbox`?

Reply:
Yes, updated.